entrypoint.sh

```diff
@@ -42,7 +40,14 @@ fi
 
 ### GEOUPDATEIP
 echo "LAUNCH GEOIP UPDATE"
-/usr/bin/geoipupdate --verbose --output --database-directory="${GEOIPUPDATE_DB_DIR}"
+if [ "${GEOIPUPDATE_DRYRUN-default}" == "default" ]; then #if GEOIPUPDATE_DRYRUN is not set
```

Comment:
```suggestion
if [ "${GEOIPUPDATE_DRYRUN:-true}" != "false" ]; then
```

I would invert the condition: if the variable is undefined, then it is dry run by default. Safer approach